hopper/flash_bwd_preprocess_kernel.h

```diff
@@ -17,13 +17,16 @@ namespace flash {
 
 using namespace cute;
 
-template <class TileShape_MK_, class Element, class ElementAccum, bool Clear_dQaccum, bool Varlen>
+template <class TileShape_MK_, class Element, class ElementAccum, class ArchTag_, bool Clear_dQaccum, bool Varlen>
 class FlashAttnBwdPreprocess {
 
 public:
 
     // Type Aliases
     using TileShape_MK = TileShape_MK_;
+    using ArchTag = ArchTag_;
+
```

Comment:
This kernel doesn't need SM90. It needs SM75 (if fp16), SM80 (if bf16), and SM89 (if fp8). Is there an easy way to rewrite this check depending on type?

Reply:
How does this look? It seems to work for me. I added it to the other kernel as well, assuming that has the same compute capability requirements.